skfda/exploratory/visualization/representation.py

```diff
@@ -119,63 +121,96 @@ def _get_color_info(
     return sample_colors, patches
 
 
-class GraphPlot:
+class GraphPlot(BasePlot):
     """
-    Class used to plot the FDataGrid object graph as hypersurfaces.
+    Class used to plot the FDatGrid object graph as hypersurfaces.
 
     When plotting functional data, we can either choose manually a color,
     a group of colors for the representations. Besides, we can use a list of
     variables (depths, scalar regression targets...) can be used as an
     argument to display the functions wtih a gradient of colors.
-
     Args:
         fdata: functional data set that we want to plot.
-        gradient_values: list of real values used to determine the color
-            in which each of the instances will be plotted.
+        gradient_color_list: list of real values used to determine the color
+            in which each of the instances will be plotted. The size
         max_grad: maximum value that the gradient_list can take, it will be
-            used to normalize the ``gradient_values``. If not
+            used to normalize the gradient_color_list in order to get values
+            thatcan be used in the funcion colormap.__call__(). If not
             declared it will be initialized to the maximum value of
             gradient_list
         min_grad: minimum value that the gradient_list can take, it will be
-            used to normalize the ``gradient_values``. If not
+            used to normalize the gradient_color_list in order to get values
+            thatcan be used in the funcion colormap.__call__(). If not
             declared it will be initialized to the minimum value of
             gradient_list.
-
+        chart (figure object, axe or list of axes, optional): figure over
+            with the graphs are plotted or axis over where the graphs are
+            plotted. If None and ax is also None, the figure is
+            initialized.
+        fig (figure object, optional): figure over with the graphs are
+            plotted in case ax is not specified. If None and ax is also
+            None, the figure is initialized.
+        axes (axis object, optional): axis over where the graphs
+            are plotted. If None, see param fig.
+        n_rows (int, optional): designates the number of rows of the figure
+            to plot the different dimensions of the image. Only specified
+            if fig and ax are None.
+        n_cols(int, optional): designates the number of columns of the
+            figure to plot the different dimensions of the image. Only
+            specified if fig and ax are None.
+    Attributes:
+        gradient_list: normalization of the values from gradient color_list
+            that will be used to determine the intensity of the color
+            each function will have.
     """
 
     def __init__(
         self,
         fdata: FData,
-        gradient_values: Optional[Sequence[float]] = None,
+        gradient_color_list: Union[Sequence[float], None] = None,
```

Comment:
Why the name and type change?

Reply:
the name for clarity, the type i will go back to the other one

Reply:
It is not more clear. They are scalar values, not colors.